smarts/core/scenario.py

```diff
@@ -52,12 +52,14 @@
 class Start:
     position: Tuple[int, int]
     heading: Heading
+    from_front_bumper: Optional[bool] = True
```

Comment:
This seems overly specific to me since we are now explicit about the vehicle feature in relation to the "start line". Common formats I remember are: front bumper, center, center of gravity, and back axle. As is, this is an interface change that will require a future change.

I think to go through it will require a new issue at minimum.

Reply:
Yeah, I agree with this being less than ideal.  (And you can look at the PR that I originally opened and then rejected for an even more less-than-ideal solution!)  So I'm definitely open to suggestions as to alternatives you think are better!

However, re it being an interface change, I have a couple of thoughts related to that:
- it's technically not an interface change because it's optional (and defaults to True, which is what it was before)so no existing users will have to change their code.
- but even so, recall this is already for the `0.5` branch (which has significant interface changes)

Reply:
Do we want to just scrap the `Start` class altogether and switch `Mission` over to use `Pose` instead?